nncf/experimental/common/tensor_statistics/statistics.py

```diff
@@ -24,6 +24,22 @@ class TensorStatistic:
 
     TENSOR_STATISTIC_OUTPUT_KEY = "tensor_statistic_output"
 
+    def get_data(self):
+        return {key: getattr(self, key) for key in self.keys()}
+
+    def load_data(self, data: Dict):
+        for key in self.keys():
+            setattr(self, key, data.get(key))
```

Comment:
How about use the same name convention as in `COMPRESSED_MODULES` and `SingleConfigQuantizationSetup`:
`get_config(self)` and `from_config(self, config)` there?
CC: @alexsu52